src/v8/isolate_scope.rs

```diff
@@ -82,17 +141,54 @@ impl<'isolate> V8IsolateScope<'isolate> {
     pub(crate) fn new_dummy(isolate: &'isolate V8Isolate) -> V8IsolateScope<'isolate> {
         V8IsolateScope {
             isolate,
-            inner_handlers_scope: std::ptr::null_mut(),
-            inner_isolate_scope: std::ptr::null_mut(),
+            storage: V8IsolateScopeStorage::default(),
         }
     }
 
+    /// Returns a raw pointer to the isolate.
+    pub fn get_raw_isolate(&self) -> *mut v8_isolate {
```

Comment:
Here again, I would prefer to avoid API's that returns the raw pointers. We have it on redismodule-rs and we see its a big no no.